apps/src/applab/commands.js

```diff
@@ -1481,10 +1464,8 @@ applabCommands.createRecord = function (opts) {
 };
 
 applabCommands.handleCreateRecord = function(opts, record) {
-  // Ensure that this event was requested by the same instance of the interpreter
-  // that is currently active before proceeding...
-  if (opts.onSuccess && opts.JSInterpreter === Applab.JSInterpreter) {
-    Applab.JSInterpreter.queueEvent(opts.onSuccess, [record]);
+  if (opts.onSuccess) {
+    opts.onSuccess.call(null, record);
```

Comment:
See bullets 3-5 in the top level description. The big difference is that opts.onSuccess is now a native function. Previously, it was an interpreter function, so the API code needed to know the convoluted method to call an interpreter function (using `queueEvent`)